### PR TITLE
DO NOT MERGE-RHOARDOC-1664: Updates to the upstream publishing process.

### PR DESCRIPTION
**Pull request by huntley**

### CODE REVIEW

File: docs/topics/proc_releasing-launcher-docs.adoc [hunk 1]
```diff
@@ -39,7 +39,35 @@ Replace `$REMOTE` with the name of the upstream remote.
 --
 
 . Request publication:
-.. File a pull request in the link:https://github.com/openshiftio/saas-launchpad/blob/master/launchpad-services/launcher-documentation.yaml#L2[openshiftio/saas-launchpad] repository, where you change the `hash` value to the hash of the commit you want to publish from the link:{link-repo-docs}[{name-repo-docs}] repository. Usually, this will be the latest commit in the `master` branch.
+.. Fork link:https://access.redhat.com/documentation/en-us/red_hat_jboss_web_server/5.0/html/red_hat_jboss_web_server_for_openshift/index[openshiftio/saas-launchpad].
+.. Clone your fork:
++
+[source,bash]
+----
+git clone git@github.com/${GIT_USER}/openshiftio/saas-launchpad.git
```
Comment (tradej) [inline docs/topics/proc_releasing-launcher-docs.adoc]: In the contrib guide, we use the `$VARIABLE_NAME` convention.

Applicable everywhere.

File: docs/topics/proc_releasing-launcher-docs.adoc [hunk 2]
```diff
@@ -39,7 +39,35 @@ Replace `$REMOTE` with the name of the upstream remote.
...
+git remote add upstream git@github.com:${GIT_USER}/openshiftio/saas-launchpad.git
+----
+.. Fetch upstream.
+.. Create a topic branch:
++
+[source,bash]
+----
+git checkout -v ${BRANCH_NAME} upstream/master
+----
+.. Edit the SHA in `launcher-documentation.yaml` on the topic branch and push your change to origin:
```
Comment (ncbaratta) [inline docs/topics/proc_releasing-launcher-docs.adoc]:
```suggestion
.. Edit the SHA in `launcher-documentation.yaml` on the topic branch and push your change to origin.
```

File: docs/topics/proc_releasing-launcher-docs.adoc [hunk 3]
```diff
@@ -39,7 +39,35 @@ Replace `$REMOTE` with the name of the upstream remote.
...
++
+The SHA value is from the commit you want to publish from the link:{link-repo-docs}[{name-repo-docs}] repository. Usually, this is the latest commit in the `master` branch
```
Comment (ncbaratta) [inline docs/topics/proc_releasing-launcher-docs.adoc]:
```suggestion
The SHA value is from the commit you want to publish from the link:{link-repo-docs}[{name-repo-docs}] repository. Usually, this is the latest commit in the `master` branch.
```

File: docs/topics/proc_releasing-launcher-docs.adoc (hunk 1 above)
Comment (huntley) [reply docs/topics/proc_releasing-launcher-docs.adoc]: Updated variables to match convention.

**Comment by inoxx03** [comment]
[test]

File: docs/topics/proc_releasing-launcher-docs.adoc [hunk 4]
```diff
@@ -39,7 +39,37 @@ Replace `$REMOTE` with the name of the upstream remote.
 --
 
 . Request publication:
-.. File a pull request in the link:https://github.com/openshiftio/saas-launchpad/blob/master/launchpad-services/launcher-documentation.yaml#L2[openshiftio/saas-launchpad] repository, where you change the `hash` value to the hash of the commit you want to publish from the link:{link-repo-docs}[{name-repo-docs}] repository. Usually, this will be the latest commit in the `master` branch.
+.. Fork link:https://access.redhat.com/documentation/en-us/red_hat_jboss_web_server/5.0/html/red_hat_jboss_web_server_for_openshift/index[openshiftio/saas-launchpad].
```
Comment (inoxx03) [inline docs/topics/proc_releasing-launcher-docs.adoc]: This does not seem like the link to the saas-launchpad repo on GitHub. It seems to point to JWS 5.0 docs.

File: docs/topics/proc_releasing-launcher-docs.adoc [hunk 5]
```diff
@@ -39,7 +39,37 @@ Replace `$REMOTE` with the name of the upstream remote.
...
+git remote add upstream git@github.com:$GIT_USER/openshiftio/saas-launchpad.git
+----
+.. Fetch upstream.
+.. Create a topic branch:
++
+[source,bash]
+----
+git checkout -v $BRANCH_NAME upstream/master
+----
+.. Edit the SHA in `launcher-documentation.yaml` on the topic branch and push your change to origin.
```
Comment (inoxx03) [inline docs/topics/proc_releasing-launcher-docs.adoc]:
```suggestion
.. Edit the SHA in link:https://github.com/openshiftio/saas-launchpad/blob/master/launchpad-services/launcher-documentation.yaml#L2[`launcher-documentation.yaml`] on the topic branch and push your change to origin.
```

File: docs/topics/proc_releasing-launcher-docs.adoc [hunk 6]
```diff
@@ -39,7 +39,37 @@ Replace `$REMOTE` with the name of the upstream remote.
...
++
+The SHA value is from the commit you want to publish from the link:{link-repo-docs}[{name-repo-docs}] repository. Usually, this is the latest commit in the `master` branch.
++
+[source,bash]
+----
+$ git push origin/$BRANCH_NAME
+----
+.. File a pull request in the link:https://github.com/openshiftio/saas-launchpad/blob/master/launchpad-services/launcher-documentation.yaml#L2[openshiftio/saas-launchpad] repository.
```
Comment (inoxx03) [inline docs/topics/proc_releasing-launcher-docs.adoc]:
```suggestion
.. File a pull request in the link:https://github.com/openshiftio/saas-launchpad[openshiftio/saas-launchpad] repository.
```

File: docs/topics/proc_releasing-launcher-docs.adoc (hunk 4 above)
Comment (inoxx03) [inline docs/topics/proc_releasing-launcher-docs.adoc]:
```suggestion
.. Fork link:https://github.com/openshiftio/saas-launchpad[openshiftio/saas-launchpad].
```
Te current URL does not seem to link to the `saas-launchpad` repo on GitHub. It seems to point to JWS 5.0 docs.

File: docs/topics/proc_releasing-launcher-docs.adoc (hunk 5 above)
Comment (huntley) [reply docs/topics/proc_releasing-launcher-docs.adoc]: @inoxx03 Do you know what the #L2 at the end of your suggested fix means?

File: docs/topics/proc_releasing-launcher-docs.adoc (hunk 6 above)
Comment (huntley) [reply docs/topics/proc_releasing-launcher-docs.adoc]: Would have taken this fix too if not for that pesky #L2 :-)

File: docs/topics/proc_releasing-launcher-docs.adoc (hunk 4 above)
Comment (huntley) [reply docs/topics/proc_releasing-launcher-docs.adoc]: Thanks!

Comment (huntley) [reply docs/topics/proc_releasing-launcher-docs.adoc]: Thanks!

**Comment by inoxx03** [comment]
This PR can not be completed due to low priority and lack of team resources. It will be closed and  can be reopened later.

**Pull request closed without merging**